Makefile

```diff
@@ -9,3 +9,8 @@ on-cluster: ## Run dev-toolbox on the cluster.
 	kubectl apply -f hack/resources.yaml
 	kubectl wait --for=condition=ContainersReady --timeout=60s pod/dev-toolbox
 	kubectl exec -it dev-toolbox -- bash
+
+.PHONY: generate-local-devcontainer
+generate-local-devcontainer: ## generate .devcontainer with local Dockerfile
```

Comment:
I would suggest renaming this target to the `local-devcontainer` to use it like this:

```
make local-devcontainer
```